returnn/tensor/tensor_dict.py

```diff
@@ -79,6 +79,7 @@ def as_raw_tensor_dict(
         include_scalar_dyn_sizes: bool = True,
         exclude_duplicate_dims: bool = False,
         expected_value_type: Union[Type, Sequence[Type]] = object,
+        exclude_sequence_lengths: bool = False,
```

Comment:
Do not add this here.

Reply:
`as_raw_tensor_dict` is specifically only intended for the purpose to add seq lens. Otherwise the whole function is not necessary at all.